tutorials/cp-starter-integration-cpi-onboard-subscribe/cp-starter-integration-cpi-onboard-subscribe.md

```diff
@@ -1,79 +1,98 @@
+[ACCORDION-BEGIN [Step 1: ](Enter your trial account)]
 
-    ![Access trial account](1.0.0.create-cp-subaccount.png)
+1. In your web browser, open the [SAP Cloud Platform trial cockpit](https://cockpit.hanatrial.ondemand.com/).
 
-    You see a subaccount with name **trial** already created. Select this subaccount.
+2. Navigate to the trial global account by clicking **Enter Your Trial Account**.
 
-    ![Access subaccount](1.0.1.access-subaccount.png)
+    !![Trial global account](01_Foundation20Onboarding_Home.png)
 
-2. In your subaccount, select **Subscriptions** > **Process Integration**.
+    >If this is your first time accessing your trial account, you'll have to configure your account by choosing a region (select the region closest to you). Your user profile will be set up for you automatically.  
+
+    >Wait till your account is set up and ready to go. Your global account, your subaccount, your organization, and your space are launched. This may take a couple of minutes.
+
+    >Choose **Continue**.
+
+    >![Account setup](02_Foundation20Onboarding_Processing.png)
+
+3. From your global account page, choose the `trial` tile to access your subaccount.
+
+
+4. Select **Subscriptions** > **Process Integration**.
 
     ![Select process integration service](1.1.select-process-integration.png)
 
-3. Select the **Subscribe** button.
+5. Select **Subscribe**.
 
     ![Select subscribe button](1.2.select-subscribe.png)
 
-    Wait for the process to complete. You will see the status change to **Subscribed** in green.
+    Wait for the process to complete. The status changes to **Subscribed** in green.
 
-    ![Subscription confirmation for Process Integratiin](1.2.1.pi-subscribed-confirmation.png)
+    ![Subscription confirmation for Process Integration](1.2.1.pi-subscribed-confirmation.png)
 
 [DONE]
 [ACCORDION-END]
 
 [ACCORDION-BEGIN [Step 2: ](Assign required roles)]
+>**IMPORTANT**:
+ With this step, you define permissions for users accessing SAP Cloud Platform Integration. For different tasks, different role templates are predefined, each role template containing the permissions for a set of tasks done by a typical persona such like the integration developer. We propose that you define an own role template that contains the most important role templates predefined by SAP.
+
+
 1. In the breadcrumbs, select your subaccount to navigate to the subaccount level.
 
     ![Navigate to subaccount](2.1.navigate-to-subaccount.png)
 
-2. Select **Role Collections** > **New Role Collection**.
+2. Select **Security** > **Role Collections** > **New Role Collection**.
 
     ![Create role collection](2.2.create-role-collection.png)
 
 3. In the **Name** field, enter **`Tenant_Administrator`** and choose **Save**. Select the newly created **`Tenant_Administrator`** role collection.
+   This newly defined role collection will be  used to group all role templates predefined by SAP that you like to assign.
```

Comment:
Pleae remove the additional space between "be" and "used".

Reply:
Hi @MichaelCzcz, 

I removed space.